### PR TITLE
fix: cui-radio-bar removal

### PR DESCRIPTION
**Pull request by ganeshkumar1989**
**SCDC-772**
### Requirements

* The cui-radio-bar component has to be replaced by oui-radio-toggle-group from ovh-ui-angular

### Description of the Change

The cui-radio-bar component has been replaced with oui-radio-toggle-group from ovh-ui-angular

### Benefits

The component from the library is re-used, giving a more uniform look across Manager

### CODE REVIEW

**Comment by ganeshkumar1989** [comment]
Thanks :)

File: client/app/dbaas/logs/detail/aliases/link/aliases-link.html [hunk 1]
```diff
@@ -9,13 +9,12 @@ <h6 data-ng-if="!ctrl.alias.loading" class="oui-header_6">
     </h6>
     <p class="oui-paragraph mb-5" data-translate="logs_aliases_link_description"></p>
     <form novalidate name="ctrl.form" class="ml-1">
-        <cui-radio-bar data-ng-model="ctrl.selectedContent">
-            <cui-radio-bar-item data-ng-repeat="content in ctrl.contents"
-                data-text="content.name | translate"
+        <oui-radio-toggle-group data-model="ctrl.selectedContent">
+            <oui-radio data-ng-repeat="content in ctrl.contents"
+                data-text="{{ content.name | translate }}"
```
Comment (FredericEspiau) [inline client/app/dbaas/logs/detail/aliases/link/aliases-link.html]: `::`

File: client/app/iplb/frontends/iplb-frontends-edit.html [hunk 2]
```diff
@@ -15,13 +15,14 @@
         <div class="oui-field">
             <label for="protocol" class="oui-label" data-translate="iplb_frontend_add_protocol"></label>
             <div class="oui-field__content">
-                <cui-radio-bar data-ng-model="ctrl.protocol"
+                <oui-radio-toggle-group data-model="ctrl.protocol"
                     data-on-change="ctrl.onProtocolChange()">
-                    <cui-radio-bar-item data-ng-repeat="protocol in ctrl.protocols"
-                        data-text="('iplb_frontend_add_protocol_' + protocol) | translate"
+                    <oui-radio data-ng-repeat="protocol in ctrl.protocols"
+                        data-text="{{ ('iplb_frontend_add_protocol_' + protocol) | translate }}"
```
Comment (FredericEspiau) [inline client/app/iplb/frontends/iplb-frontends-edit.html]: `::`

File: client/app/iplb/graph/iplb-graph.html [hunk 3]
```diff
@@ -1,12 +1,13 @@
...
     <cui-page-content-title text="'iplb_graph_title' | translate"></cui-page-content-title>
 
-    <cui-radio-bar data-ng-model="ctrl.scale"
+    <oui-radio-toggle-group data-model="ctrl.scale"
         data-on-change="ctrl.onScaleChange()">
-        <cui-radio-bar-item data-ng-repeat="(scale, value) in ctrl.scales"
-            data-text="('iplb_graph_scape_' + scale) | translate"
-            data-value="scale"></cui-radio-bar-item>
-    </cui-radio-bar>
+        <oui-radio data-ng-repeat="(scale, value) in ctrl.scales"
```
Comment (FredericEspiau) [inline client/app/iplb/graph/iplb-graph.html]: `track by`

File: client/app/iplb/frontends/iplb-frontends-edit.html [hunk 4]
```diff
@@ -15,13 +15,14 @@
         <div class="oui-field">
             <label for="protocol" class="oui-label" data-translate="iplb_frontend_add_protocol"></label>
             <div class="oui-field__content">
-                <cui-radio-bar data-ng-model="ctrl.protocol"
+                <oui-radio-toggle-group data-model="ctrl.protocol"
                     data-on-change="ctrl.onProtocolChange()">
-                    <cui-radio-bar-item data-ng-repeat="protocol in ctrl.protocols"
-                        data-text="('iplb_frontend_add_protocol_' + protocol) | translate"
+                    <oui-radio data-ng-repeat="protocol in ctrl.protocols"
```
Comment (FredericEspiau) [inline client/app/iplb/frontends/iplb-frontends-edit.html]: `track by`

File: client/app/dbaas/logs/detail/aliases/link/aliases-link.html [hunk 5]
```diff
@@ -9,13 +9,12 @@ <h6 data-ng-if="!ctrl.alias.loading" class="oui-header_6">
     </h6>
     <p class="oui-paragraph mb-5" data-translate="logs_aliases_link_description"></p>
     <form novalidate name="ctrl.form" class="ml-1">
-        <cui-radio-bar data-ng-model="ctrl.selectedContent">
-            <cui-radio-bar-item data-ng-repeat="content in ctrl.contents"
-                data-text="content.name | translate"
+        <oui-radio-toggle-group data-model="ctrl.selectedContent">
+            <oui-radio data-ng-repeat="content in ctrl.contents"
```
Comment (FredericEspiau) [inline client/app/dbaas/logs/detail/aliases/link/aliases-link.html]: `track by`

File: client/app/dbaas/logs/detail/aliases/link/aliases-link.html (hunk 1 above)
Comment (ganeshkumar1989) [reply client/app/dbaas/logs/detail/aliases/link/aliases-link.html]: This is done :)

File: client/app/iplb/frontends/iplb-frontends-edit.html (hunk 2 above)
Comment (ganeshkumar1989) [reply client/app/iplb/frontends/iplb-frontends-edit.html]: This is fixed

File: client/app/iplb/graph/iplb-graph.html (hunk 3 above)
Comment (ganeshkumar1989) [reply client/app/iplb/graph/iplb-graph.html]: This is done :)

File: client/app/iplb/frontends/iplb-frontends-edit.html (hunk 4 above)
Comment (ganeshkumar1989) [reply client/app/iplb/frontends/iplb-frontends-edit.html]: This is done :)

File: client/app/dbaas/logs/detail/aliases/link/aliases-link.html (hunk 5 above)
Comment (ganeshkumar1989) [reply client/app/dbaas/logs/detail/aliases/link/aliases-link.html]: This is fixed :)

File: client/app/oui-less-override.less [hunk 6]
```diff
@@ -46,3 +46,13 @@
 .tui-dropdown-menu {
     z-index: 10;
 }
+
+.oui-radio-toggle-group {
+    // Overidden to avoid text wrapping in radio component.
```
Comment (whispyy) [inline client/app/oui-less-override.less]: poke @AxelPeter : we should consider it in ui-kit.

**Pull request merged**